app/src/main/java/com/example/calculator/Calculation.java

```diff
@@ -5,13 +5,57 @@
 public class Calculation {
 
     private ArrayList<Token> mStringArrayList;
+    private double answer;
```

Comment:
Add a prefix 'm' for the member variables.
And apply the rule throughout the file.